src/Action/IndexCreator.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+
+namespace oat\tao\elasticsearch\Action;
+
+
+use common_report_Report as Report;
+use Exception;
+use oat\oatbox\extension\script\ScriptAction;
+use oat\tao\elasticsearch\ElasticSearch;
+use oat\tao\model\search\Search;
+
+class IndexCreator extends ScriptAction
```

Comment:
If we want to remove tao-core coupling for this lib, do you thing it is good to create another tao-core script here?